packages/enzyme/src/Utils.js

```diff
@@ -21,9 +21,25 @@ export function getAdapter(options = {}) {
 }
 
 export function makeOptions(options) {
+  const { attachTo, hydrateIn } = options;
+
+  if (attachTo && hydrateIn && attachTo !== hydrateIn) {
```

Comment:
would there ever be a case when both are present?  Do we just want to not allow this and then hydrate -> attach on pre16 versions?

Reply:
So, I’ve normalized the options here - which means it will end up having both the best time it hits this code path. So altho users really shouldn’t supply both, i cant think of how to differentiate between user options and things passed in via shallow calls or adapter options etc.